.github/workflows/rust.yml

```diff
@@ -81,7 +81,7 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: test
-          args: -Zbuild-std=std --target x86_64-unknown-linux-gnu --features=exts
+          args: -Zbuild-std=std --target x86_64-unknown-linux-gnu --features=exts --package uefi --package uefi-macros
```

Comment:
I'm guessing a rebase will replace this with `./build.py test`, from #283

Reply:
Yep, rebased to use `build.py test`.